assessment.css

```diff
@@ -0,0 +1,42 @@
+body {
+    background-color: lavender;
+   
+    width: 100vh;
```

Comment:
Your website is not responsive at all, mainly due to the fact that you specified the body width to be in `vh` units which are relative to the height of the browser window (viewport). Check what happens with you website as you vary the height of the browser window.